mysql-test/include/percona_partition_with_compressed_columns.inc

```diff
@@ -0,0 +1,30 @@
+# ==== Purpose ====
+#
+# Common test to check support for compressed columns on partitioned tables
+# On multiple Storage Engines.
+#
+# ==== Usage ====
+#
+# --let $engine= ENGINE
+# --source include/partition_with_compressed_columns.inc
```

Comment:
```suggestion
# --source include/percona_partition_with_compressed_columns.inc
```

Reply:
@laurynas-biveinis good catch. Ready for next review